packages/js/src/components/contentAnalysis/mapResults.js

```diff
@@ -106,9 +106,10 @@ export function getIconForScore( score ) { // eslint-disable-line complexity
 		case "ok":
 			return { icon: "seo-score-ok", color: colors.$color_ok };
 		case "bad":
-			return { icon: "seo-score-bad", color: colors.$color_red };
+		// No break or return statement here because the icon for bad and default scores is the same.
+		// eslint-disable-next-line no-fallthrough
```

Comment:
I think the code would still work if you remove the `bad` case. Since it's the same as the default, it will receive the `seo-score-bad` icon by reaching the default case.

Reply:
Yes, absolutely true. I left it because it's easier to understand what the code does in this case. But perhaps it's more appropriate to include it as a comment instead?

Reply:
Yeah, a comment instead sounds good!